icu4c/source/i18n/unicode/ulistformatter.h

```diff
@@ -43,25 +43,23 @@ struct UFormattedList;
 typedef struct UFormattedList UFormattedList;
 #endif /* U_HIDE_DRAFT_API */
 
-#ifndef U_HIDE_DRAFT_API
 /**
  * FieldPosition and UFieldPosition selectors for format fields
  * defined by ListFormatter.
- * @draft ICU 63
+ * @stable ICU 63
  */
 typedef enum UListFormatterField {
```

Comment:
Keep this draft if we're not promoting the format(FieldPositionIterator) method yet.  I think it will need to go stable next cycle when formatToValue goes stable though.

Reply:
got it

Reply:
Did you make this change?  Keep it draft.

Reply:
constants are still showing promoted to stable, will add comment on .md file